source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -251,12 +343,6 @@ void BioXASAppController::updateGenericScanConfigurationDetectors()
 	}
 }
 
-void BioXASAppController::initializeBeamline()
```

Comment:
I make it a pure virtual function, which forces the subclass to implement it.

It is a misleading to have an implementation for BioXASBeamline, since in fact, we must instance the BioXASSideBeamline or BioXASMainBeamline.

A side note about removing this is that this implementation caused two hours yesterday to figure out why my codes doesn't work :-1: